locales/en.default.json

```diff
@@ -215,7 +236,22 @@
             "product_search_empty": "No products found",
             "product_search_progress": "Searching...",
             "categories": "Categories",
-            "products": "Products"
+            "products": "Products",
+            "product_added_request": "Product is added to quote request",
```

Comment:
`product_added_request`, `go_request`, `update_request` move to `quotes.general`. `update_request` may replace `quotes.general.update_quote` (it's nowhere used). Move `cart.general.close_cart` and `cart.general.countinue_shop` to `dialog.close` and `dialog.continure_shop`. Replace `products.compare_list.close_dialog` with `dialog.close`